src/test/java/lesson8Person/PersonTest.java

```diff
@@ -0,0 +1,43 @@
+package lesson8Person;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class PersonTest {
```

Comment:
Generalnie tego typu testy nie są nam potrzebne. Starajmy się mieć otestowaną głównie logikę biznesową, ale z racji, że dopiero się uczymy jest to jak najbardziej w porządku.